product_docs/docs/edbcloud/beta/migration/cold_migration.mdx

```diff
@@ -0,0 +1,177 @@
+      ```
+      CREATE ROLE postgres;
+      ALTER ROLE postgres WITH SUPERUSER ….;
+      ```
+    2. Remove any role with superuser or replication privileges. For example, remove lines like these:
+       ```
+       CREATE ROLE admin;
+       ALTER ROLE admin WITH SUPERUSER ….;
+       ```
+    3. For every role that is created, grant the new role to the `edb_admin` user immediately after the creation of the user. For example:
+       ```
+       CREATE ROLE my_role;
+       GRANT my_role TO edb_admin;
+       ```
+    4. Remove the `NOSUPERUSER`, `NOCREATEROLE`, `NOCREATEDB`, `NOREPLICATION`, `NOBYPASSRLS` permission attributes on the other users.
+
+   The role section in the modified file should look similar to:
+   ```
+   CREATE ROLE my_role;
+   GRANT my_role TO edb_admin;
+   ALTER ROLE my_role WITH INHERIT LOGIN PASSWORD 'SCRAM-SHA-256$4096:my-Scrambled-Password';
+   ```
+1. From the migration host, execute:
+
+   ```
+   psql -1 -f roles.sql -d "postgres://edb_admin@pg-target:5432/edb_admin?sslmode=require"
+   ```
+   (Replace `pg-target` with the Fully Qualified Domain Name (FQDN) of your EDB Cloud cluster).
+
+
+   This command tries to create the roles in a single transaction. In case of errors, the transaction is rolled back, leaving the database cluster in the same state as before the import attempt. This behavior is enforced using the `-1` option of `psql`.
+
+## Exporting a database
+From the migration host, use the `pg_dump` command to export the source database into the target database in EDB Cloud. For example:
+
+```
+pg_dump -Fc -d "host=pg-source user=postgres dbname=app" -f app.dump
+```
+
+!!! Note
+    You can use the `--verbose` option to monitor the progress of the operation.
+
+The command generates a custom .dump archive (`app.dump` in this example), which contains the compressed dump of the source database. The duration of the command execution varies depending on several variables including size of the database, network speed, disk speed, and CPU of both the source instance and the migration host. You can inspect the table of contents of the dump with `pg_restore -l <db_name>.dump`.
+
+As with any other custom format dump produced with `pg_dump`, you can take advantage of the features that `pg_restore` provides you with, including:
+- Selecting a subset of the import tasks by editing the table of contents and passing it to the `-L` option.
+- Running the command in parallel using the `-j` option in conjunction with the directory format.
+
+For further information, see the [`pg_restore` documentation](https://www.postgresql.org/docs/current/app-pgrestore.html).
+
+## Importing a database
+Use the `pg_restore` command and the .dump file you creating when exporting the source database to import the database into EDB Cloud. For example:
+
+```
+pg_restore -C -d  "postgres://edb_admin@pg-target:5432/edb_admin?sslmode=require" app.dump
```

Comment:
Done

Reply:
@ba-edb @mlewandowski-edb , I changed three instances of the sslmode parameter to verify-full in this topic. Is that correct?

Reply:
I agree that `verify-full` is the correct option for production; however, it will require more setup on the client machine related to the certificates.  For non-production environments, especially those for evaluation use cases, this may not already have been done.  If we default to using `verify-full` in our example connection strings, we should at least note that `require` is an option.  Since we want to guide our customers to using `verify-full`, then we should provide links to documentation related to setting up the environment to support using it.

Reply:
@ba-edb @mlewandowski-edb, I'm wondering if we should go back to -require since that is what is shown in the Quick connect string in the portal itself. 
When you reply, can you tag me? Even though I am watching this PR I am not getting notified when you comment on it. Thanks.

Reply:
@drothery-edb: I'm fine with that - all good points here.